### ISSUE TITLE
Latest streamdeck has issues with CodePathMac/CodePathWin

### ISSUE THREAD

**Issue by viraptor** [FritzAndFriends/StreamDeckToolkit#186]
**Describe the bug**

I used the template to generate a new plugin. Tried to run it with Streamdeck 6.3.0.18948, but the plugin was never loaded in the list.

No errors in the Streamdeck logs or windows event log. With procmon I noticed that Streamdeck tries to open / read from `....sdPlugin\MarkResult`, `....sdPlugin\MarkResult.html` which were not referenced in my manifest. (I expected `...\MarkResult.exe`)

While trying to align the `manifest.json` with the latest one from `net.voicemod.windowsdesktop.sdPlugin`, I noticed they list only one `CodePath`, but the template created `CodePathWin` and `CodePathMac`. After removing those and using `CodePath` instead, the plugin is immediately visible and working as expected.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Streamdeck: 6.3.0.1894

**Comment by AlexHedley** [comment]
Is this similar to https://github.com/FritzAndFriends/StreamDeckToolkit/issues/158

**Comment by viraptor** [comment]
You're right, it seems like a duplicate, I just didn't find it at the time 🤷
